src/Command/Db/DbSizeCommand.php

```diff
@@ -78,19 +82,40 @@ protected function execute(InputInterface $input, OutputInterface $output) {
         $machineReadable = $table->formatIsMachineReadable();
         $showInBytes = $input->getOption('bytes') || $machineReadable;
 
-        $columns  = ['max' => 'Allocated disk', 'used' => 'Estimated usage', 'percent_used' => '% used'];
+        $columns  = ['max' => 'Allocated disk', 'used' => 'Estimated usage', 'percent_used' => 'Percentage used'];
         $values = [
             'max' => $showInBytes ? $allocatedDisk : Helper::formatMemory($allocatedDisk),
-            'used' => $showInBytes ? $estimatedUsage : Helper::formatMemory($estimatedUsage),
+            'used' => $showInBytes ? $estimatedUsage['__TOTAL__'] : Helper::formatMemory($estimatedUsage['__TOTAL__']),
             'percent_used' => $this->formatPercentage($percentageUsed, $machineReadable),
         ];
-
         $this->stdErr->writeln('');
         $table->render([$values], $columns);
-
+        
         $this->showWarnings($percentageUsed);
+                
+
 
-        $this->showInaccessibleSchemas($service, $database);
+        $db_table   = $this->getService('table');
+        $db_columns = ['db' => 'Database', 'used' => 'Estimated usage', 'percent' => 'Percentage'];
```

Comment:
Having two tables and naming the columns in both is going to be awkward, I believe it will break the `--columns` option.

Reply:
You're right. It does produce weird results. I moved that logic under a flag to make it optional.